templates/.kokoro/build.sh

```diff
@@ -23,11 +23,44 @@ cd ${scriptDir}/..
 # include common functions
 source ${scriptDir}/common.sh
 
+# TODO: Switch monorepo_script_output to main
+# Find the root commit between the two branches
+base_commit=$(git merge-base monorepo_script_output HEAD)
```

Comment:
Kokoro pipeline is in the PR branch, so HEAD so point to the latest commit in this branch. Should this compared with the branch it is trying to be merged into instead of always `main`/`monorepo_script_output`?

Reply:
Hardcoding monorepo_script_output is fine for now.